app/src/main/java/com/p2p/presentation/lobby/ServerLobbyFragment.kt

```diff
@@ -16,21 +15,23 @@ abstract class ServerLobbyFragment<GVM : GameViewModel> : BaseGameFragment<
         ServerLobbyViewModel,
         GVM>() {
 
-
-    override val viewModel: ServerLobbyViewModel by viewModels()
-
     override val gameInflater: (LayoutInflater, ViewGroup?, Boolean) -> FragmentServerLobbyBinding =
         FragmentServerLobbyBinding::inflate
 
     private lateinit var connectedPlayersAdapter: ConnectedPlayersAdapter
 
+    protected abstract val continueText: String
+    protected abstract val continueAction: () -> Unit
+    protected open val onConnectedPlayers: (List<String>) -> Unit = { _ -> }
```

Comment:
Estas dos cosas son métodos, no atributos 😅 

```suggestion
    protected abstract fun continueAction()
    protected open fun onConnectedPlayers(players: List<String>) {}
```